web/usuaris/src/locales/en.json

```diff
@@ -145,19 +124,17 @@
   "Aceptar": "Accept",
   "Bloquear": "Block",
   "Desbloquear": "Unblock",
+  "administrador": "Admin",
+  "registrado": "Registered",
+  "mentor": "Mentor",
```

Comment:
The keys here have to be the same in all 3 languages, otherwise it won't translate. In the catalan ones is "ADMIN", "REGISTERED" and "MENTOR"